view/template/dashboard.hbs

```diff
@@ -225,7 +229,63 @@
         data: data,
         options: options
     });
+
     var legend = document.getElementById('legend');
+    var table = $('#trials').DataTable();
+    var selectedVal = 'Pending';
+    var date = new Date();
+    var startDate = new Date();
+    var endDate = new Date();
 
     legend.innerHTML += radarChart.generateLegend();
+
+    $('.dropdown-menu').on('click', 'a', function () {
+        'use strict';
+        selectedVal = $(this).text();
```

Comment:
`var selectedVal  = $(this).text().trim();` should work out any quirks in the string.
The set the value directly `$('#selectedChoice').val(selectedVal);`